test/fbEmulator.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+import os
+import pygame
+import io
+
+fifo_path = "/tmp/emulatedFramebuffer"
+mode = 0o600
+
+try:
+    os.mkfifo(fifo_path, mode=mode)
+except OSError as e:
+    print("FIFO creation error:", e)
+    print("Continuing.")
+
+real_bit_depth=[5,6,5]
+
+bytes_per_pixelgroup=int(sum(real_bit_depth)/8)
+
+fake_bit_depth=8
+
+
+#sliced_x, sliced_y = (6480,3600)
+#zoom_ratio=1
+
+sliced_x, sliced_y = (192,108)
```

Comment:
Just as a question, I'm guessing this is referring to 1080p through a 10x upscaling?

Reply:
Yeah, for ease of testing on a regular ol' monitor.